assets/javascripts/discourse/components/modal/recalculate-scores-form.hbs

```diff
@@ -0,0 +1,64 @@
+<DModal
+  class="recalculate-scores-form-modal"
+  @title={{i18n "gamification.recalculate"}}
+>
+  <:body>
+    {{#if (eq this.status "loading")}}
+      <div class="status">{{i18n "gamification.recalculating"}}</div>
+    {{else if (eq this.status "complete")}}
+      <div class="status">{{d-icon "check"}}
+        {{i18n "gamification.completed"}}</div>
+    {{else}}
+      <form class="form-horizontal">
+        <div class="input-group">
+          <div>{{i18n "gamification.update_scores_help"}}</div>
```

Comment:
Can we add a class name here, `recalculate-modal__lead-text` (we use BEM for naming) so we can target it for styling.